stack-manager/src/main/descriptor/vertx-stack-full.json

```diff
@@ -104,6 +104,12 @@
       "version": "\${vertx.version}",
       "included": false
     },
+    {
+      "groupId": "io.vertx",
+      "artifactId": "vertx-monitoring-influxdb",
+      "version": "\${vertx.version}",
+      "included": false
```

Comment:
@vietj , not sure if we keep using the dropwizard impl as the default or not.

Reply:
yes we keep it, it works out of the box + it provides JMX support. Also it would be a breaking change.